tests/core/consensus/test_clique_consensus.py

```diff
@@ -247,7 +247,7 @@ def test_import_block(paragon_chain):
 
     signed_header = sign_block_header(vm.get_block().header.copy(
         extra_data=VANITY_LENGTH * b'0' + SIGNATURE_LENGTH * b'0',
-        state_root=b'\x99\xaa\xf5CF^\x95_\xce~\xe4)\x00\xb1zr\x1dr\xd6\x00N^\xa6\xdc\xc41\x90~\xb7te\x00',  # noqa: E501
+        state_root=b'\x02g\xd5{\xf9\x9f\x9e\xab)\x06\x1eY\x9a\xb7W\x95\xfd\xae\x9a:\x83m%\xbb\xcc\xe1\xca\xe3\x85d\xa7\x05',  # noqa: E501
```

Comment:
Did this change come from `ethereum/tests`?

Reply:
It actually comes from [this change](https://github.com/ethereum/py-evm/pull/2080/files#diff-7e062b3a842e7d3a8c2d2db83993b89b2003282531a858e9b6ae2a2559bdb95aL368-R385). The coinbase account is touched even when block reward is zero which affects the state root.